frontend/src/metabase/visualizations/echarts/cartesian/option/axis.ts

```diff
@@ -32,6 +32,16 @@ export const getAxisNameGap = (ticksWidth: number): number => {
   return ticksWidth + CHART_STYLE.axisNameMargin;
 };
 
+const parseNumberValue = (value: any): number | null => {
```

Comment:
Having a copy of this function here is worse than just extracting it into a shared location and reuse here and in other components, wdyt?